README.md

```diff
@@ -582,28 +628,51 @@ hero:
 
 ### Adding a new blog post 📝
 
-1. Create a new feature branch. Example: `feature/new-hello-world-post`.
-2. Add post feature and hero images to `apps/labs/public/posts/<post-name>`.
-3. Add a new `.md|.mdx` file inside the `app/labs/posts` directory. Make sure to read the
-   [Structure of the blog post section](#structure-of-the-blog-post) in this file to ensure
-   that the post is properly structured.
-4. Commit and push your changes to the repository. For commits please follow the format of the conventional commit.
-   [See](https://www.conventionalcommits.org/en/v1.0.0/)
-5. Create a pull request to the `develop` branch. Make sure to add the `type: content 📝` and `labs 🔭` labels to the PR.
-6. Wait for someone in the website team to review the new blog post. If everything is ok, the PR will be merged to the `develop` branch.
+> **Note** If there is a deadline or other timing consideration for the
+> publication of your post, **please coordinate actively** with the Marketing
+> team (currently @Katebrack and @bskinn) and the website ops team
+> (#qwebsite-ops channel on Slack) from as early as possible to ensure we can
+> meet your requirements.
+
+1. Create a new feature branch off of the `develop` branch. Example:
+   `<site>-blog/new-hello-world-post`.
+2. Add post images (feature, hero, and any for the post body) to
+   `apps/<site>/public/posts/<post-name>`.
+3. Add a new `.md|.mdx` file inside the `app/<site>/posts` directory. Make sure
+   to read the
+   [Structure of the blog post section](#structure-of-the-blog-post) in this
+   file to ensure that the post is properly structured.
+4. Commit and push your changes to the repository. For commits please follow the
+   format of the conventional commit. See the
+   [Conventional Commits docs](https://www.conventionalcommits.org/en/v1.0.0/).
+5. Create a pull request to the `develop` branch. Make sure to add the
+   `type: content 📝`, `owner: Quansight`, and either `labs 🔭` or
+   `consulting 🤝` labels to the PR.
+6. Wait for someone on the website team to review the new blog post. If
+   everything is ok, the PR will be merged to the `develop` branch. The blog
+   post will then go live with the next site build (merge to `main`).
```

Comment:
```suggestion
   post will then go live with the next production deployment (merge to `main`).
```

Since "production deployment" is the language that Vercel uses